frb_codegen/Cargo.toml

```diff
@@ -56,3 +56,5 @@ x86_64-pc-windows-msvc = {pkg-fmt = "zip"}
 chrono = []
 default = ["chrono"]
 uuid = []
+# When disabled, name qualifiers are discarded, e.g. my_library::SyncReturn == flutter_rust_bridge::SyncReturn
```

Comment:
uber-nit: curious why do we want that? IMHO nobody wants my_library::SyncReturn to be misunderstood as frb's SyncReturn

Reply:
I see, the old behavior does so. Good new behavior!